covid-vue-app/test/frontend-test.js

```diff
@@ -4,10 +4,10 @@ const assert = require("assert");
 const driver = new Builder().forBrowser("firefox").build();
 var numberOfTestsCompleted = 0;
```

Comment:
Consider using `let` instead of `var`
```suggestion
let numberOfTestsCompleted = 0;
```

Reply:
Scoping reasons [here](https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var)